### DISCUSSION TITLE
Where is Deck.gl?

### DISCUSSION THREAD

**Discussion by riazarbi** [apache-superset/superset-ui#1332] [Questions]
Hello,

We rely quite heavily on deck.gl map integration with superset. In fact, I've started poking around this repo to figure out how so add an H3HexagonLayer deck.gl viz to superset. 

However, a quick code search via the github search bar shows no code (except one readme note) that references the work `deck`. 

I see that the superset repo file `/superset-frontend/src/visualizations/presets/MainPreset.js` imports 

`import { DeckGLChartPreset } from '@superset-ui/legacy-preset-chart-deckgl';`

But when I come back to this repo to find that reference directory I can't find it anywhere. 

Forgive me, I am not a node programmer and don't understand the build system very well. 

But I wanted to ask - is the plan to keep including deck.gl charts into the future? Or does this lack of references suggest that it will be removed in future releases?

**Comment by villebro** [comment]
Hi @riazarbi ! It's here: https://github.com/apache-superset/superset-ui-plugins-deckgl

**Comment by riazarbi** [comment]
Thanks @villebro ! I’ll head over to that repo to find out how to include the H3HexagonLayer
